desktop/apps/auction/routes.coffee

```diff
@@ -48,7 +48,6 @@ setupUser = (user, auction) ->
 
   artworks.comparator = (artwork) ->
     saleArtwork = artwork.related().saleArtwork
-    saleArtwork.get('lot_number') or
```

Comment:
I don't think we should be sorting by `lot_label` at all? The whole reason we have `position` is so we can determine lot order without caring about what the `lot_label` is.